sdk/identity/identity/CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 # Release History
 
+## 1.0.3 (Unreleased)
+
+- Fix tracing to set correct span attributes ([PR link](https://github.com/Azure/azure-sdk-for-js/pull/6565)).
```

Comment:
The word `link` in the link text doesnt provide any additional value. Using the PR number on the other hand does.
```suggestion
- Fix tracing to set correct span attributes ([PR 6565](https://github.com/Azure/azure-sdk-for-js/pull/6565)).
```

Reply:
On second thoughts, isnt the fix to use correct span attributes made in `core-http` and not the `identity` library?

Reply:
It has one small change to the span kind
https://github.com/azure/azure-sdk-for-js/commit/0c3bf8da8f#diff-e7eb1f6c97c7fd6b89b1ffe8cd3c31fc